GLTFSDK/Source/MeshPrimitiveUtils.cpp

```diff
@@ -560,45 +556,32 @@ namespace
             throw GLTFException("Input triangulated triangle fan has fewer than 3 indices.");
         }
 
-        const auto triangleCount = indexCount / 3;
-        const auto resultIndexCount = 2 + triangleCount;
-        std::vector<T> result(resultIndexCount);
-
-        result[0] = indices[0];
-        result[1] = indices[1];
-        result[2] = indices[2];
+        std::vector<T> result;
```

Comment:
You need to call reserve on this now that we're using push_back.

Reply:
Done.